elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -132,11 +138,10 @@ class PfeContentSet extends PFElement {
   connectedCallback() {
     super.connectedCallback();
 
-    if (this.hasLightDOM()) {
-      Promise.all([customElements.whenDefined(PfeTabs.tag), customElements.whenDefined(PfeAccordion.tag)]).then(() => {
-        this._build();
-      });
-    }
+    // If the element has an ID, postfix container
+    if (this.id) this.id = `${this.id}-container`;
```

Comment:
@heyMP I was coding last night and remembered we moved the content set into the light DOM to resolve an issue with duplicate IDs so I added some logic to postfix the container and heading IDs.